pylint/config/option_manager_mixin.py

```diff
@@ -274,7 +274,10 @@ def read_config_file(self, config_file=None, verbose=None):
             self.set_current_module(config_file)
             parser = self.cfgfile_parser
             if config_file.endswith(".toml"):
-                self._parse_toml(config_file, parser)
+                try:
+                    self._parse_toml(config_file, parser)
+                except toml.TomlDecodeError as e:
+                    self.add_message("parse-error", line=0, args=str(e))
```

Comment:
I don't think `parse-error` should be used here. The description of the error refers to `astroid` not being able to build a representation of the code.

Reply:
Very true, I'm glad you reviewed that ! https://github.com/PyCQA/pylint/pull/5365/commits/b43206f0bbb27bdf3f09f13371d667f44d5781f6